src/sidebar/components/SidebarView.tsx

```diff
@@ -132,6 +132,11 @@ function SidebarView({
   return (
     <div className="relative">
       <h2 className="sr-only">Annotations</h2>
+      {pendingUpdatesNotification && (
+        <div className="fixed z-1 right-2 top-12">
+          <PendingUpdatesNotification />
+        </div>
+      )}
```

Comment:
This element is position fixed, so it renders the same regardless of where it is among the rest of its parent's children.

However, placing it higher, also makes it be earlier in the focus sequence.